mock matchers › failing output › expect.mock.toReturnTimes threw exception
<dim>expect.mock(</dim><red>mock</red><dim>).toReturnTimes(</dim><green>expected</green><dim>)</dim>

Expected the mock to return <green>one time</green>, but it didn't return.
